lib/themes.dart

```diff
@@ -701,6 +701,7 @@ class Palette {
     required this.onPrimaryLight,
     Color? onPrimaryOpacity7,
     Color? onPrimaryOpacity10,
+    Color? onPrimaryOpacity20,
     Color? onPrimaryOpacity25,
```

Comment:
Opacity of 20% looks almost exactly like opacity of 25%, thus can't we use `onPrimaryOpacity25` instead of creating a new color? Will this affect the visuals?

Reply:
@egorozh, I guess it doesn't look distinguishable to the end user. I suggest to use `onPrimaryOpacity25` here